spec/01-unit/01-db/01-schema/02-metaschema_spec.lua

```diff
@@ -501,3 +502,219 @@ describe("metaschema", function()
     assert.truthy(MetaSchema:validate(MetaSchema))
   end)
 end)
+
+
+describe("metasubschema", function()
+  it("rejects a bad schema", function()
+    local s = {
+      name = "bad",
+      fields = {
+        { foo = "bar", },
+      },
+      primary_key = { "foo" },
+    }
+    assert.falsy(MetaSchema.MetaSubSchema:validate(s))
```

Comment:
I think we should test the error messages as well (not the whole returned table, maybe just the relevant entry - `assert.equals("some error", err.fields.foo)`